18 Normalizr/src/model/api/student.ts

```diff
@@ -3,12 +3,14 @@ export class Student {
   gotActiveTraining: boolean;
   fullname: string;
   email: string;
+  countryId: number;
 
   public constructor() {
     this.id = -1;
     this.gotActiveTraining = false;
     this.fullname = "";
     this.email = "";
+    this.countryId = -1;
```

Comment:
I think null can be a good value, on the other hand is it a good idea to define this as a class or should it be an interface? Should we add a factory to initialize a new instance?